src/reporter.js

```diff
@@ -1,52 +1,74 @@
-const fs = require('fs');
-const path = require('path');
-const readFiles = require('read-files-promise');
-const mustache = require('mustache');
-const utils = require('./utils.js');
-const chalk = require('chalk');
-const getPort = require('get-port');
-const WebSocket = require('ws');
-
-const REPORT_FILENAME = path.join('smashtest', 'report.html');
-const REPORT_DATA_FILENAME = path.join('smashtest', 'report-data.js');
-const PASSED_DATA_FILENAME = path.join('smashtest', 'passed-data');
-const SMASHTEST_SS_DIR = path.join('smashtest', 'screenshots');
+const fs = require("fs");
+const path = require("path");
+const readFiles = require("read-files-promise");
+const mustache = require("mustache");
+const utils = require("./utils.js");
+const chalk = require("chalk");
+const getPort = require("get-port");
+const WebSocket = require("ws");
+
+let REPORT_FILENAME,
+    REPORT_DATA_FILENAME,
+    PASSED_DATA_FILENAME,
+    SMASHTEST_SS_DIR,
+    initialFolder,
+    folder;
+
+const date = Date.now();
 
 /**
  * Generates a report on the status of the tree and runner
  */
 class Reporter {
-    constructor(tree, runner) {
-        this.tree = tree;               // the Tree object to report on
-        this.runner = runner;           // the Runner object to report on
+  constructor(tree, runner) {
+    this.tree = tree; // the Tree object to report on
+    this.runner = runner; // the Runner object to report on
 
-        this.reportTemplate = "";       // template for html reports
-        this.reportTime = null;         // Date when the report was generated
+    this.reportTemplate = ""; // template for html reports
+    this.reportTime = null; // Date when the report was generated
 
-        this.isReportServer = true;     // whether or not to run the report server
-        this.reportDomain = null;       // domain:port where report server's api is available
-        this.wsServer = null;           // websocket server object
+    this.isReportServer = true; // whether or not to run the report server
+    this.reportDomain = null; // domain:port where report server's api is available
+    this.wsServer = null; // websocket server object
 
-        this.prevSnapshot = null;       // previous snapshot sent over websockets
+    this.prevSnapshot = null; // previous snapshot sent over websockets
 
-        this.timerFull = null;          // timer that goes off when it's time to do a full write
-        this.timerSnapshot = null;      // timer that goes off when it's time to do a snapshot write
+    this.timerFull = null; // timer that goes off when it's time to do a full write
+    this.timerSnapshot = null; // timer that goes off when it's time to do a snapshot write
```

Comment:
I want to make tabs = 4 spaces, for consistency :)

Reply:
Also, the comments should remain aligned horizontally

Reply:
Ok sorry is a Linter auto format :)